engine/table/src/main/java/io/deephaven/engine/table/impl/indexer/DataIndexer.java

```diff
@@ -155,16 +164,31 @@ public boolean hasDataIndex(@NotNull final Collection<ColumnSource<?>> keyColumn
      * @param keyColumnNames The key column for which to retrieve a DataIndex
      * @return The {@link DataIndex}, or {@code null} if one does not exist
      */
-    public static DataIndex getDataIndex(@NotNull Table table, final String... keyColumnNames) {
-        if (keyColumnNames.length == 0) {
+    @Nullable
+    public static DataIndex getDataIndex(@NotNull final Table table, final String... keyColumnNames) {
+        return getDataIndex(table, Arrays.asList(keyColumnNames));
+    }
+
+    /**
+     * If {@code table} has a DataIndexer, return a {@link DataIndex} for the given key columns, or {@code null} if no
+     * such index exists, if the cached index is invalid, or if the {@link DataIndex#isRefreshing() refreshing} cached
+     * index is no longer live.
+     *
+     * @param table The {@link Table} to check
+     * @param keyColumnNames The key column for which to retrieve a DataIndex
```

Comment:
Can you fix this error on my part?
```suggestion
     * @param keyColumnNames The key columns for which to retrieve a DataIndex
```